modules/azure/synapse.go

```diff
@@ -0,0 +1,86 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/synapse/mgmt/2020-12-01/synapse"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// GetSynapseWorkspaceClientE is a helper function that will setup a mysql server client.
+func GetSynapseWorkspaceClientE(subscriptionID string) (*synapse.WorkspacesClient, error) {
```

Comment:
GetSynapseWorkspaceClientE this helper does not seem to be used anywhere! can you clean it up

Reply:
Thanks @HadwaAbdelhalem cleaned up